src/lib/y2partitioner/widgets/md_description.rb

```diff
@@ -0,0 +1,81 @@
+require "cwm/widget"
+
+Yast.import "HTML"
+
+require "y2partitioner/widgets/blk_device_attributes"
+require "y2partitioner/widgets/help"
+
+module Y2Partitioner
+  module Widgets
+    # Widget that is richtext filled with description of md raid passed in constructor
+    class MdDescription < CWM::RichText
+      include Yast::I18n
+      include Help
+
+      # @param md [Y2Storage::Md] to describe
```

Comment:
it is not yet in storage-ng, just wrapped from arvins home, so I cannot add versioned dependency.